### PR TITLE
Make organization redirect warning more clear

### PR DESCRIPTION
**Pull request by n0toose**

### CODE REVIEW

**Comment by n0toose** [comment]
Changes the following warning:

image

This change was made after a recent situation of a certain set of users not reading the red letter warning that pops up. Maybe the wording could be improved?

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -2545,7 +2545,7 @@ settings.visibility.private_shortname = Private
 
 settings.update_settings = Update Settings
 settings.update_setting_success = Organization settings have been updated.
-settings.change_orgname_prompt = Note: changing the organization name also changes the organization's URL.
+settings.change_orgname_prompt = Note: Changing the organization name will also change your organization's URL and free the old name.
```
Comment (fogti) [inline options/locale/locale_en-US.ini:2548]: why is `Changing` capitalized?

Comment (n0toose) [reply options/locale/locale_en-US.ini:2548]: Why wouldn't it be? It makes things look more consistent (this is how every other string looks like) and it also doesn't "break the flow".

Comment (wxiaoguang) [reply options/locale/locale_en-US.ini:2548]: https://www.grammarly.com/blog/capitalization-after-colons/

It is right.

**Review by wxiaoguang — APPROVED** [review]

**Review by lunny — APPROVED** [review]

File: options/locale/locale_en-US.ini [hunk 2]
```diff
@@ -640,7 +640,7 @@ update_language_success = Language has been updated.
 update_profile_success = Your profile has been updated.
 change_username = Your username has been changed.
 change_username_prompt = Note: username changes also change your account URL.
```
Comment (fogti) [inline options/locale/locale_en-US.ini:642]: > Why wouldn't it be?

because it is then inconsistent with this entry.

**Pull request merged**